src/util/TestCaseExcelExportUtil.ts

```diff
@@ -29,9 +124,29 @@ export const createExcelExportDtosForAllTestCases = (
 ): TestCaseExcelExportDto[] => {
   const testCaseExcelExportDtos: TestCaseExcelExportDto[] = [];
 
+  let groupNumber = 1;
   measure.groups?.forEach((group) => {
     const testCaseExecutionResultDtos: TestCaseExecutionResultDto[] = [];
-    measure.testCases?.forEach((currentTestCase) => {
+    const testCasesByGroup: TestCase[] = getTestCasesByGroup(measure, group.id);
+    testCasesByGroup?.forEach((currentTestCase) => {
+      const groupPopulations = currentTestCase.groupPopulations?.filter(
+        (groupPopulation) => {
+          return groupPopulation.groupId === group.id;
```

Comment:
Ideally filter should only return 1 groupPopulation. so can we say groupPopulation instead ? we could avoid iterating over the groupPopulations as well.